control/control.go

```diff
@@ -325,6 +325,8 @@ func (c *Controller) Status(req *controlapi.StatusRequest, stream controlapi.Con
 	eg.Go(func() error {
 		return c.solver.Status(ctx, req.Ref, ch)
 	})
+	emtpyLogVertex := controlapi.VertexLog{}
+	emtpyLogVertexSize := emtpyLogVertex.Size()
```

Comment:
you can make this global and move setting it to `init()`

Reply:
fixed